### ISSUE TITLE
[Bug]: Could not initialize NNPACK! Reason: Unsupported hardware.

### ISSUE THREAD

**Issue by hunarjain09** [xserrat/docker-facebook-demucs#23]
### Contact Details

huja4688@colorado.edu

### What happened?

I am trying to run the amazing demucs-software on my Apple M1 Pro using the docker image but getting the following error.

### In which operating system you found the issue?

MacOS

### Relevant log output

```shell
Selected model is a bag of 4 models. You will see that many progress bars per track.
Separated tracks will be stored in /tmp/output/mdx_extra_q
Separating track /tmp/d93bdecf9dd25d29394238327db07b0fabfdeaba860ec3bd3e2c341d3d0ea0ca.mp3

  0%|                                                                                   | 0.0/33.0 [00:00<?, ?seconds/s][W NNPACK.cpp:79] Could not initialize NNPACK! Reason: Unsupported hardware.
```

**Comment by hunarjain09** [comment]
Is it possible to build the docker image that supports ARM 64.
We have been given an Assignment which uses this image to separate songs and we don't have enough CPU to build the image it would be really helpful :)

**Comment by xserrat** [comment]
Thanks @hunarjain09 to open this bug. 

Right now, the docker image built only supports amd64 cpu architecture but it's something I would like to tackle very soon.

Meanwhile, feel free to open a pull request, we'll be very welcomed! 🙌🏽

I'll let you know as soon as I get the fix 💪🏼

**Comment by xserrat** [comment]
Hi @hunarjain09 , thanks to @chucknelson we have this pull request that adds support to ARM 64.

We'll merge it very soon ;)

**Comment by xserrat** [comment]
Closing issue after merging the fix #24 . Thanks for reporting @hunarjain09 !
